components/api/src/main/java/com/hotels/styx/api/FormData.java

```diff
@@ -70,10 +70,10 @@
     /**
      * Helper method to access data of a single parameter.
      *
-     * @param key
+     * @param key parameter key
      * @return The parameter value wrapped in an {@link Optional} if the parameter exists, {@link Optional#empty} otherwise.
      */
-    public Optional<String> postParam(String key) {
+    Optional<String> postParam(String key) {
```

Comment:
`FormData` is part of the API. We cannot reduce visibility just because the internal code is not using it from outside the package.